packages/bridge/src/toast/toast.types.ts

```diff
@@ -0,0 +1,17 @@
+import { BridgeRequest } from "../messaging";
+import { BridgeMethod } from "../types";
+import { SwirlToastIntent } from "@getflip/swirl-components/loader";
+
+export type ShowToastRequestParams = {
+  content: string;
+  duration?: number;
+  icon?: string;
```

Comment:
This would require a mapper in the mobile app to map from the icon name to the corresponding `SwirlIcons`- `IconData` type.
We need to decide if we do it, or if we drop it for now also.

Reply:
Maybe we can just use the glyph names of our icons. These should be the same in Swirl and Flip Mobile. Ignoring stuff like camelCase vs kebap-case, of course.

Reply:
That was my idea, just wanted to raise awareness for the mobile guys that we would maybe require a new mapper/parser in the mobile app, similar to the `flip_legacy_icon_parser`,to map from the glyph names to the `IconData`, because `flip_legacy_icon_parser` only does that for what we call "symbols" in swirl terminology.

Reply:
Yes we would need something like `flip_legacy_icon_parser` to support icons.